### PR TITLE
Manage state via Redux for Team members and save data in JSON Server

### PR DESCRIPTION
**Pull request by RikFrieling**

### CODE REVIEW

File: components/team-member-form/TeamMemberForm.jsx [hunk 1]
```diff
@@ -90,7 +130,7 @@ class TeamMemberForm extends React.Component {
                     name="lastName"
                     id="last-name"
                     required
-                    pattern="([A-Za-z\\s-]+)"
+                    onChange={(event) => this.handleChange(event)}
```
Comment (jellebouwman) [inline components/team-member-form/TeamMemberForm.jsx]: You could write this like `onChange={this.handleChange}`

File: components/header/header-nav-bar/header-nav-bar.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    margin-top: 70px;
+    z-index: 20;
+
+    @include desktop {
+      align-items: center;
+      flex-direction: row;
+      height: auto;
+      margin-top: 0;
+      padding-top: 16px;
+      }
```
Comment (boris-arkenaar) [inline components/header/header-nav-bar/header-nav-bar.scss]: Indenting is off

File: components/header/header-nav-bar/header-nav-bar.scss [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+  &__item {
+    color: $color-letter-on-primary;
+    font-family: $font-primary;
+    font-size: $font-size-mobile;
+    font-stretch: normal;
+    font-style: normal;
```
Comment (boris-arkenaar) [inline components/header/header-nav-bar/header-nav-bar.scss]: These `normal`s can probably go.

File: components/header/header-nav-bar/header-nav-bar.scss [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    margin-bottom: 43px;
+    margin-right: 14px;
+    padding: 0 18px 18px;
+
+    @include desktop {
+      font-size: $font-size-primary;
+      font-weight: $font-weight-light;
+      margin-bottom: 0;
+      padding: 8px 18px 18px;
+      }
```
Comment (boris-arkenaar) [inline components/header/header-nav-bar/header-nav-bar.scss]: Indenting is off.

File: components/header/header-nav-bar/header-nav-bar.scss [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  & a {
+    text-decoration: none;
+
+    &:visited {
+      color: $color-monochrome-1;
+    }
+  }
+  }
```
Comment (boris-arkenaar) [inline components/header/header-nav-bar/header-nav-bar.scss]: Indenting is off.

File: components/header/header.scss [hunk 6]
```diff
@@ -32,18 +40,30 @@
 
   &--open {
     .menu-toggle__item {
-      &--open {
+      &-cross {
```
Comment (boris-arkenaar) [inline components/header/header.scss]: Only nest `__` elements and `--` modifiers. Write this as
```
.menu-toggle__item-cross {
  display: none;
}
.menu-toggle__item-hamburger {
  display: block;
}
```

File: components/header/header.scss [hunk 7]
```diff
@@ -162,14 +111,15 @@
...
-  margin-right: 31px;
-  @include tablet {
-    display: block;
+    .arrow-down {
+      margin-right: 10px;
+      padding: 15px;
+      position: absolute;
+      right: 0;
+      top: 10;
+
```
Comment (boris-arkenaar) [inline components/header/header.scss]: Unnecessary empty lines.

File: components/header/header.scss [hunk 8]
```diff
@@ -162,14 +111,15 @@
...
+
   }
 }
+}
```
Comment (boris-arkenaar) [inline components/header/header.scss]: Indenting is off.

File: components/team-members-overview/team-members-overview.scss [hunk 9]
```diff
@@ -1,18 +1,89 @@
...
+  &__header {
+    display: flex;
+
+    @include tablet {
+      align-items: baseline;
+      justify-content: space-between;
+      margin-top: 29px;
+    }
+
+    &-buttons {
```
Comment (boris-arkenaar) [inline components/team-members-overview/team-members-overview.scss]: Don't nest, define it as `&__header-buttons`.

**Pull request merged**